deepforest/_layer.py

```diff
@@ -135,10 +135,7 @@ def fit_transform(self, X, y, sample_weight=None):
         n_samples, self.n_features = X.shape
 
         X_aug = []
-        if self.is_classifier:
```

Comment:
I think there is no need to use the attribute `is_classifier` any more. Could you remove it?